### DISCUSSION TITLE
Retraining Timeseries data

### DISCUSSION THREAD

**Discussion by KornkamonMat** [jdb78/pytorch-forecasting#1706] [Q&A]
Hi! I'm using the TFT model with the TimeSeriesDataSet for time series forecasting and would like to retrain the model using data from the past six months. How can I ensure that the categorical encoders applied to the new data will produce the same encoder values as before?

Thanks in advance!

**Comment by fkiraly** [comment]
Interesting question. Is that currently not ensured?

Could you perhaps post some example code for the retraining, so we can check what happens under the hood?
